extensions/arc/package.json

```diff
@@ -719,9 +751,11 @@
                           "variableName": "AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_NAME",
                           "type": "text",
                           "description": "%arc.postgres.server.group.name.validation.description%",
-                          "textValidationRequired": true,
-                          "textValidationRegex": "^[a-z]([-a-z0-9]{0,10}[a-z0-9])?$",
-                          "textValidationDescription": "%arc.postgres.server.group.name.validation.description%",
+                          "validations" : [{
+                            "type": "regex_match",
+                            "regex": "^[a-z]([-a-z0-9]{0,10}[a-z0-9])?$",
+                            "description": "%arc.postgres.server.group.name.validation.description%"
+                          }],
                           "required": true
                         },
                         {
```

Comment:
Are we able to validate that a number must be an interger (no decimals)?  We'd want that for AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_WORKERS.